src/serlo-editor-repo/store/focus/reducer.ts

```diff
@@ -1,4 +1,5 @@
 import * as R from 'ramda'
+import * as Reselect from 'reselect'
```

Comment:
This was just a quick workaround because there is already a `createSelector` function defined that would result in a name clash. I agree with your suggestion.

Reply:
`import {creatSelector as reselectCreateSelector} from 'reselct'` :)

Reply:
I would rather remove or rename our custom `createSelector` function. But I'll wait with that for now so that there aren't too may changes.